### DISCUSSION TITLE
Maddalena Arrow Wan 2 (module RF for water meter) - which rtl_433 parameters ?

### DISCUSSION THREAD

**Discussion by stan1m** [merbanan/rtl_433#2713] [Q&A]
Hello,
I am trying to read information from Arrow Wan 2 with these parameters : rtl_433 -f 868.95M -s 1200k  (protocol id 104).
But I have no result (I can read devices on 433 mhz with only "rtl_433" so there is no hardware or config issue).
I use usb stick with fc0012 chip (on raspberry pi). The distance sender-receiver is about 15 meters.
Thanks for your help.
Stan
(France)
https://www.maddalena.it/en/product/arrow-wan-2/

**Comment by zuckschwerdt** [comment]
Try with adding `-Y autolevel -M level -M noise` and also `-v` or `-vv` or selectivly `-R 104:vv` to see more messages.

**Reply by stan1m** [reply]
Thanks a lot ! With your additionnal parameters I get now a few lines for Wireless M-Bus, Mode C&T, 100kbps.
I suppose the device (arrow wan 2) is not fully supported yet, because the lines are not readable (different lenghts of codes, unformatted).
How could I get a comprehensible result ?

pi@pi-b:~ $ rtl_433 -f 868.95M -s 1200k -Y autolevel -M level -M noise -R 104:vv
rtl_433 version -129-NOTFOUND branch master at 202311011021 inputs file rtl_tcp RTL-SDR

New defaults active, use "-Y classic -s 250k" if you need the old defaults
Found Fitipower FC0012 tuner
[SDR] Using device 0: Realtek, RTL2838UHIDIR, SN: 00000001, "Generic RTL2832U OEM"
[Auto Level] Current noise level -15.6 dB, estimated noise -15.6 dB
[pulse_slicer_pcm] Wireless M-Bus, Mode C&T, 100kbps (-f 868.95M -s 1200k)
codes     : {407}0107481200820000001000080014002020047000001520004883000a005d11024e0004000419304578602205a029100a803006
codes     : {342}002008400a000000009001089140004400020000124800001a0000231cc0012a2010080000080210001004
[Auto Level] Current noise level -15.6 dB, estimated noise -15.5 dB
codes     : {718}0e00000006820080000010400000c4004a00042000000100040081000000000000024a018010802443041ac3002a02c00a20000001c01000000040000000402f4040004100002040000000180200001020018000020000000000
codes     : {339}9128050405811c105148008080000003d0800253506000000010004020000004200c2a04d600000000000
codes     : {322}1e86000080000038000000600500c0000248004010624282b900000800003de000002240004014020
codes     : {702}c86000002042000804011900240088002810229080010004008000600000005000100410402060100000c002740c600194040400246000290008000019c03441da02008252a804c013000002b00140004000008000000200
codes     : {691}00008000000000000000000000000000400000040000214404b02808080030001000002500000001b0000004004001000010000000080900ca009004000000003000140000002000000200004020180028900006a0000
codes     : {734}800000400000000004000000000c000251e458000000000504800381047408580c0030018800000101002800200000c00060000004100008000000bfb0000000004800400011304b04020800480000150d280841314010c260000404
codes     : {679}02120040040802194007c1040000500200010a380005088884002022304043404901a8004204000000000080411c000040000034800187a4020001020022401002100478080000080000002024d900408001200042
[Auto Level] Current noise level -15.7 dB, estimated noise -15.6 dB
codes     : {175}949720100940020c023405500028468c884400252304
codes     : {436}000000000000000000b40000000804000400080600c000000000092014008200120020080000000000090000400002001800080200402
^CSignal caught, exiting!

**Reply by stan1m** [reply]
Hello,
Please I need more help to start decoding frames from this hardware
https://www.maddalena.it/en/product/arrow-wan-2/
Thanks
